app/components/wallet-warning.tsx

```diff
@@ -6,14 +6,13 @@ export const WalletWarning: FC<FlexProps> = props => {
   return (
     <Flex width="100%" borderRadius="6px" mt="tight" {...props}>
       <Text textStyle="caption" color="ink.600">
-        ⚠️ We're currently resolving a number of issues related to network connectivity and Ledger
-        devices.
+        ⚠️ Please check our status page for known issues if you’re experiencing any difficulties.
```

Comment:
Will emojis render in all contexts?

Reply:
It's UTF-8 I think? Either way it should be fine.